modules/admin/proc-rhdh-deployment-config.adoc

```diff
@@ -96,6 +98,59 @@ spec:
                   claimName: dynamic-plugins-root
 ----
 
+Starting from v1alpha3 (Operator version 0.4.0), you can mount directories from pre-created PVCs using the `spec.application.extraFiles.pvcs` field. PVCs are mounted as directories to the container's path, based on the following logic:
```

Comment:
Just curious why this content is located here..
This is currently under [`Chapter 11. Configuring Red Hat Developer Hub deployment`](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-702/admin-rhdh/#proc-rhdh-deployment-config_assembly-admin-templates), which IIUC seems to talk more about the `spec.deployment.patch` Custom Resource field (introduced in the `v1alpha2` CRD version as mentioned in the chapter intro).
So I'm wondering if this should not be somewhere else (but not sure where).

Reply:
> The content is currently located here as it describes PVCs within the RHDH operator, which aligns with this section

But this whole [Chapter 11](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/administration_guide_for_red_hat_developer_hub/proc-rhdh-deployment-config_assembly-admin-templates) is more about the `spec.deployment.patch` field with a lot of different examples for different cases, but with all of them updating the same field.

![image](https://github.com/user-attachments/assets/3b11f655-f347-45f5-a80e-6a827befa341)

Reply:
But if you think it makes sense this way, let's go with that ;-)

Reply:
@rm3l, great point! However, I think the current scope for 1.4 focuses on adding the feature for mounting PVCs to the RHDH operator. For now, I can add a procedure as a section of Chapter 11, as it’s somewhat related. Once we have all the relevant content available, we could consider restructuring it—perhaps including a section on extra files.

I’ll also create a Jira ticket to ensure we capture the content related to adding extra files. What do you think?

Reply:
I added the content in a separate procedure section. 
Jira to capture the content related to adding extra files: https://issues.redhat.com/browse/RHIDP-4977